### PR TITLE
Add animations to panels

### PR DESCRIPTION
**Pull request by johnnyraper**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
     "@angular/platform-browser-dynamic": "6.0.3",
     "@angular/router": "6.0.3",
     "core-js": "2.5.7",
+    "ngx-row-accordion": "^2.1.0",
```
Comment (maxime1992) [inline package.json]: Can you revert the changes on package.json?

File: projects/ngx-row-accordion/src/lib/ngx-row-accordion.component.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+.ngx-row-accordion {
+  display: flex;
+  width: 500px;
+  overflow: hidden;
+}
+
+.title {
+  position: relative;
+  z-index: 20;
```
Comment (maxime1992) [inline projects/ngx-row-accordion/src/lib/ngx-row-accordion.component.scss]: Is that needed?

File: yarn.lock [hunk 3]
```diff
@@ -6091,6 +6091,12 @@ ng-packagr@3.0.1:
     uglify-js "^3.0.7"
```
Comment (maxime1992) [inline yarn.lock]: Revert those changes too please

File: package.json (hunk 1 above)
Comment (johnnyraper) [reply package.json]: Reverted file to previous version.

File: yarn.lock (hunk 3 above)
Comment (johnnyraper) [reply yarn.lock]: Reverted file to previous version.

File: projects/ngx-row-accordion/src/lib/ngx-row-accordion.component.scss (hunk 2 above)
Comment (johnnyraper) [reply projects/ngx-row-accordion/src/lib/ngx-row-accordion.component.scss]: Hey Maxime, yes this is needed to ensure the panel remains on top and visible throughout the animation. I've added a comment in the code to explain and reduced the z-index value down. 👍

**Comment by maxime1992** [comment]
As discussed IRL this is not really working and I'm closing for now. Feel free to reopen if you feel like it might be fixed.

**Comment by johnnyraper** [comment]
Thanks for the heads up @maxime1992 👍
